HISTORY.rst

```diff
@@ -2,10 +2,15 @@
 History
 =======
 
+0.2.2
+-----
+* Set wcs.getCoverage timeout to 120 seconds.
+* Fix `Raven.parse_results` logic when no flow observations are present and no diagnostic file is created.
+* Fix ECCC test where input was cached and shadowed forecast input data
```

Comment:
Nitpicking but can we stick with using or not using periods (`.`) at the end of items please.

Reply:
Added periods everywhere.